### PR TITLE
Use module.exports to export default methods

### PR DESCRIPTION
**Pull request by majapw**
Also export named exports as individual files

to: @ljharb @lencioni @airbnb/webinfra

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "repository": "airbnb/react-with-styles-interface-css",
   "homepage": "https://github.com/airbnb/react-with-styles-interface-css#readme",
   "bugs": "https://github.com/airbnb/react-with-styles-interface-css/issues",
-  "main": "dist",
+  "main": "index.js",
```
Comment (ljharb) [inline package.json]: this is fine, but `./` or `index` are better, in that order :-)

**Comment by ljharb** [comment]
Technically this is a breaking change; but we could also do a v1.0.1 and deprecate v1.0.0 if we consider this a bug

**Comment by majapw** [comment]
It's probably fine to just do a v2, no?

**Comment by ljharb** [comment]
Yup!

**Pull request merged**